README.md

```diff
@@ -52,6 +52,17 @@ go get github.com/mongodb/awscdk-resources-mongodbatlas-go/awscdkresourcesmongod
 
 ## Available Constructors
 MongoDB Atlas AWS CDK Resources provides L1, L2 and L3 CDK constructors. Please, have a look at [README.md](src/README.md#cdk-constructors) for a full lists of constructors and examples.
+
+## Limitations
+1. [Resource import](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-import.html) is not supported for third-party resources.
+
+2. Any third-party resource must support all CRUD operations. MongoDB Atlas for CloudFormation does not support the following MongoDB Atlas resources because they do not support all CRUD operations:
+
+* cloud-backup-snapshot-export-job
+* cloud-provider-access
+* federated-settings-identity-provider
+* federated-settings-org-configs
```

Comment:
* `Any third-party resource must support all CRUD operations` I guess this is because of AWS CFN requirements, right? Should we specify that?
* is this meant to be an exhaustive list?

Reply:
- Yes and we can, the language comes from an earlier PR we merged in (I'll go back and update if we make changes here)
- No it's not, but if we have an exhaustive list we can include it. My only concern there would be maintaining the accuracy. Alternatively I can change the language to be clear it's not exhaustive

Unrelated, I'm still blocked by this `Code Health / call-package-workflow / build (pull_request)` check that I just can't figure out. I ran `npx projen` and I've tried several variations of a clean start, but I keep getting "Files changed during build." Do you/ does anyone else on the team know how I might solve it? Thanks, sorry for the headache.

Reply:
> Yes and we can, the language comes from an earlier PR we merged in (I'll go back and update if we make changes here)

Ok, I would suggest to do so in order to be more clear with customers.

> Alternatively I can change the language to be clear it's not exhaustive

yes, suggest to specify that.